<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelUTSetup</class>
 <widget class="QWidget" name="PanelUTSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="PanelChannelMgr" name="widget_ChannelMgr" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>通道号：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_ChannelNo"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTabWidget" name="tabWidgetControl">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="tabTimebase">
               <attribute name="title">
                <string>时序</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_18">
                <item row="0" column="0">
                 <widget class="PanelUTSetupTimebase" name="widgetTimebase" native="true"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tabTransmit">
               <attribute name="title">
                <string>发射</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_16">
                <item row="0" column="0">
                 <layout class="QHBoxLayout" name="layoutTransmit">
                  <item>
                   <widget class="QGroupBox" name="PRFGroup">
                    <property name="title">
                     <string>PRF</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_2">
                     <item row="1" column="0">
                      <widget class="ConfigSpinBox" name="PRFSpinBox">
                       <property name="maximum">
                        <number>8000</number>
                       </property>
                       <property name="singleStep">
                        <number>100</number>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="0">
                      <widget class="QDial" name="PRFDial">
                       <property name="maximum">
                        <number>8000</number>
                       </property>
                       <property name="singleStep">
                        <number>10</number>
                       </property>
                       <property name="pageStep">
                        <number>100</number>
                       </property>
                       <property name="notchesVisible">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="Gain">
               <attribute name="title">
                <string>增益</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_17">
                <item row="0" column="1">
                 <widget class="QGroupBox" name="gainRefGroupBox">
                  <property name="title">
                   <string>参考增益</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_7">
                   <item row="1" column="0">
                    <widget class="ConfigDoubleSpinBox" name="gainRefSpinBox">
                     <property name="maximum">
                      <double>90.000000000000000</double>
                     </property>
                     <property name="singleStep">
                      <double>0.100000000000000</double>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QDial" name="gainRefDial">
                     <property name="maximum">
                      <number>9000</number>
                     </property>
                     <property name="singleStep">
                      <number>10</number>
                     </property>
                     <property name="pageStep">
                      <number>600</number>
                     </property>
                     <property name="notchesVisible">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QGroupBox" name="gainGroupBox">
                  <property name="title">
                   <string>增益</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_6">
                   <item row="1" column="0">
                    <widget class="ConfigDoubleSpinBox" name="gainSpinBox">
                     <property name="decimals">
                      <number>2</number>
                     </property>
                     <property name="maximum">
                      <double>90.000000000000000</double>
                     </property>
                     <property name="singleStep">
                      <double>0.100000000000000</double>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QDial" name="gainDial">
                     <property name="maximum">
                      <number>9000</number>
                     </property>
                     <property name="singleStep">
                      <number>10</number>
                     </property>
                     <property name="pageStep">
                      <number>600</number>
                     </property>
                     <property name="notchesVisible">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="2">
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QGroupBox" name="rectifierModeGroup">
                    <property name="title">
                     <string>波形选择</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_9">
                     <item row="0" column="0">
                      <widget class="ConfigRadioButton" name="rectifierModeRadioButton1">
                       <property name="text">
                        <string>正半波</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="ConfigRadioButton" name="rectifierModeRadioButton3">
                       <property name="text">
                        <string>全波</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="ConfigRadioButton" name="rectifierModeRadioButton2">
                       <property name="text">
                        <string>负半波</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="ConfigRadioButton" name="rectifierModeRadioButton4">
                       <property name="text">
                        <string>射频</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="suppressGroup">
                    <property name="title">
                     <string>抑制</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_10">
                     <item row="0" column="0">
                      <widget class="ConfigDoubleSpinBox" name="suppressSpinBox"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>复制通道数据</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>源通道号：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Source"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>目标通道号：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_DestStart"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>到</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_DestEnd"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Copy">
            <property name="text">
             <string>复制</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigRadioButton</class>
   <extends>QRadioButton</extends>
   <header location="global">ConfigControl/ConfigRadioButton.h</header>
  </customwidget>
  <customwidget>
   <class>ConfigDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">ConfigControl/ConfigDoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ConfigSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">ConfigControl/ConfigSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>PanelUTSetupTimebase</class>
   <extends>QWidget</extends>
   <header>PanelUTSetup/PanelUTSetupTimebase.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanelChannelMgr</class>
   <extends>QWidget</extends>
   <header>PanelChannel/PanelChannelMgr.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>PRFDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PRFSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PRFSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PRFDial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
